src/modules/checkBox.js

```diff
@@ -2,7 +2,11 @@ window.handleCheckbox = (id) => {
   const tasks = JSON.parse(localStorage.getItem('TasksInfo')) || [];
   for (let i = 0; i < tasks.length; i += 1) {
     if (id === tasks[i].id) {
-      tasks[i].completed = true;
+      if (tasks[i].completed === true) {
+        tasks[i].completed = false;
+      } else {
+        tasks[i].completed = true;
+      }
     }
```

Comment:
- [Optional] Dear @maths29, though your functionalities are working well, I would suggest adding the strict/line through on the tasks that are completed. This makes your task more visible and easy to follow.

> Something like this...
![image](https://github.com/maths29/Todo-List/assets/69077061/575cf7ba-fb1e-41a7-8d73-70f91c445ac8)